disnake/audit_logs.py

```diff
@@ -234,6 +238,7 @@ class AuditLogChanges:
         'guild_id':                      ('guild', _transform_guild_id),
         'tags':                          ('emoji', None),
         'default_message_notifications': ('default_notifications', _enum_transformer(enums.NotificationLevel)),
+        'communication_disabled_until':  ('guild_timeout', _transform_datetime),
```

Comment:
surely a simple lambda would do the trick here

Reply:
Sure, but `_transform_color` or others are trivial as well. In theory those could all be lambdas, but having a separate func keeps it in line with the rest